src/bin/formatters/html.ts

```diff
@@ -8,25 +8,24 @@ const htmlFormatter: FormatterCallback = function (formatter) {
     fileContent += '<body>'
     fileContent += '<center><h2>Violation Report</h2></center>'
 
-    fileContent += '<table border = 1>'
+    fileContent += '<table border="1">'
     fileContent +=
       '<tr><th>Number#</th><th>File Name</th><th>Line Number</th><th>Message</th></tr>'
 
-    const arrAllMessages = event.arrAllMessages
-    arrAllMessages.forEach((fileInfo) => {
-      const arrMessages = fileInfo.messages
-      arrMessages.forEach((message, i) => {
-        fileContent = `${fileContent}<tr><td>${i + 1}</td><td>${
-          fileInfo.file
-        }</td><td>${message.line}</td><td>${message.message}</td></tr>`
-      })
-    })
+    for (const { file, messages } of event.arrAllMessages) {
+      fileContent += messages
+        .map(
+          ({ line, message }, i) =>
+            `<tr><td>${
+              i + 1
+            }</td><td>${file}</td><td>${line}</td><td>${message}</td></tr>`
+        )
+        .join('')
+    }
 
-    // TODO: replace with what?
-    // @ts-expect-error
-    fileContent = fileContent.replace('</table></body></html>')
```

Comment:
I think this part was buggy since a long time...
We should write a test to check the emitted `report.html`